frontend/src/scenes/data-warehouse/new/sourceWizardLogic.tsx

```diff
@@ -186,6 +187,66 @@ export const SOURCE_DETAILS: Record<string, SourceConfig> = {
             },
         ],
     },
+    Snowflake: {
+        name: 'Snowflake',
+        caption: (
+            <>
+                Enter your Snowflake credentials to automatically pull your Snowflake data into the PostHog Data
+                warehouse.
+            </>
+        ),
+        fields: [
+            {
+                name: 'account_id',
+                label: 'Account ID',
+                type: 'text',
+                required: true,
+                placeholder: '',
+            },
+            {
+                name: 'database',
+                label: 'Database',
+                type: 'text',
+                required: true,
+                placeholder: 'snowflake_sample_data',
+            },
+            {
+                name: 'warehouse',
+                label: 'Warehouse',
+                type: 'text',
+                required: true,
+                placeholder: 'compute_warehouse',
```

Comment:
suggestion: By default, Snowflake stores and resolves identifiers in uppercase characters. We are later quoting this, so we override the automatic uppercase resolution.

From a user perspective, they may be thinking this resolves to the default warehouse name, `COMPUTE_WAREHOUSE`, in uppercase. However, this is not the case as we later quote this. 

All in all, I'd suggest we use the default uppercase name here:
```suggestion
                placeholder: 'COMPUTE_WAREHOUSE',
```